maxim/logger/openai/completions.py

```
@@ -10,6 +10,50 @@
 from .utils import OpenAIUtils
 
 
+class StreamWrapper:
+    def __init__(self, stream, generation, trace, is_local_trace):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)